build.gradle

```diff
@@ -24,7 +24,7 @@ buildscript {
         maven { url 'https://maven.fabric.io/public' }
     }
     dependencies {
-        classpath 'com.android.tools.build:gradle:3.0.0-alpha2'
+        classpath 'com.android.tools.build:gradle:3.0.0-alpha3'
```

Comment:
@caiorrs ~~just change that field to a 3 
To alpha2 to alpha3.
If you cant figure it out I will fix it later. But please pull the latest changes. ~~

On second thought can you join us on slack and I will guide you?   

_Sent from my HTC Nexus 9 using [FastHub](https://play.google.com/store/apps/details?id=com.fastaccess.github)_